Runtime/Model/BacktraceHttpClient.cs

```diff
@@ -167,7 +167,14 @@ private void AddAttachmentToFormData(List<IMultipartFormSection> formData, IEnum
 
             foreach (var file in uniqueAttachments)
             {
-                if (string.IsNullOrEmpty(file) || File.Exists(file) == false || new FileInfo(file).Length > maximumAttachmentSize)
+                if (string.IsNullOrEmpty(file) || File.Exists(file) == false)
+                {
+                    continue;
+                }
+
+                var fileSize = new FileInfo(file).Length;

```

Comment:
This is a potential race condition - the file could be deleted after `File.Exists()` returns `true`. As improbable as it is, this is possible.

Reply:
@perf2711 we need to get a file size - we don't want to check if a file exists. The file might exist, but the file size might be 0.

Reply:
All right, I was just worried that `new FileInfo` may throw an exception, but it should not.